cmd/test-ble/main.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"os"
+	"os/signal"
+	"strings"
+
+	"github.com/teslamotors/vehicle-command/internal/log"
+	"github.com/teslamotors/vehicle-command/pkg/connector/ble"
+)
+
+var (
+	btAdapter = flag.String("btAdapter", "", "Optional ID of Bluetooth adapter to use (Linux only)")
+	testScan  = flag.Bool("testScan", false, "Also test BLE scan")
+)
+
+func main() {
```

Comment:
`make install` is going to dump `test-ble` into the users PATH, which I don't think is desirable.

Consider merging with https://github.com/teslamotors/vehicle-command/blob/main/examples/ble/main.go.

Reply:
...or add a tesla-control command that blocks until the car is detected nearby, if you believe others woudl find this helpful.